{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/assets/js/utils/external-links/index.js

```diff
@@ -0,0 +1,63 @@
+/*
+External links (defined as all links pointing to a domain that is not the
+same as the one from which the current document is being served)
+will have their 'target' attribute set to '_blank'.
+
+Social media sharing links will show in a popup.
+*/
+export default function () {
+  const links = [].slice.call(document.getElementsByTagName('a'))
```

Comment:
I'd change to to `Array.from(document.getElementsByTagName('a'))` as this is the natural API way of doing this.